github_discovery.py

```diff
@@ -314,6 +314,40 @@ def get_slack_channel_name_by_id(slack_channel_id):
   log.debug(f'Slack channel name for {slack_channel_id} is {slack_channel_name}')      
   return slack_channel_name
 
+def find_channel_by_severity_label(alert_severity_label):
+    try:
+        response = requests.get(ALERTMANAGER_URL, verify=False)
+        if response.status_code == 200:
+            alertmanager_data = response.json()
+            config_data = alertmanager_data['config']
+            formatted_config_data = config_data["original"].replace('\\n', '\n')
+            yaml_config_data = yaml.safe_load(formatted_config_data)
+            json_config_data = json.loads(json.dumps(yaml_config_data))
+
+            # Find the receiver name for the given severity
+            receiver_name = ''
+            for route in json_config_data['route']['routes']:
+                if route['match'].get('severity') == alert_severity_label:
```

Comment:
There might be cases where there are more than one route that matches here - so maybe we need to account for that.

Reply:
I have verified the data for few of them and data in all routes was exactly same for the channel